control-plane-operator/hostedclusterconfigoperator/controllers/resources/kas/admissionpolicies.go

```diff
@@ -27,16 +28,27 @@ const (
 	AdmissionPolicyNameConfig = "config"
 	AdmissionPolicyNameMirror = "mirror"
 	AdmissionPolicyNameICSP   = "icsp"
+	AdmissionPolicyNameInfra  = "infra"
+
+	BaseCelExpression = "has(object.spec) && has(oldObject.spec) && object.spec == oldObject.spec"
 )
 
 var (
+	// This is a list of users that are allowed to create, update, or delete resources
+	// without limitation in the HostedCluster side. It is usually used for the clusterOperators
+	// that needs to modify the Spec after/during the HostedCluster is created.
+	userWhiteList = []string{
+		fmt.Sprintf("system:%s", config.HCCOUser),
+		"system:serviceaccount:openshift-network-operator:cluster-network-operator",
```

Comment:
This whitelist should only be used for the infra validating policy. Other policies should only have HCCOUser.